blueocean-dashboard/src/main/js/components/karaoke/components/Pipeline.jsx

```diff
@@ -46,6 +46,15 @@ export default class Pipeline extends Component {
         this.updateOnFinish = KaraokeConfig.getPreference('runDetails.pipeline.updateOnFinish').value;
         this.stopOnClick = KaraokeConfig.getPreference('runDetails.pipeline.stopKaraokeOnAnyNodeClick').value === 'always';
         this.state = { tailLogs: false };
+
+        this.debounceFetchNodes = debounce((karaokeOut) => {
+            logger.debug('sse fetch it', this.karaoke);
+            if (karaokeOut) {
+                this.pager.fetchNodesOnly({});
+            } else {
+                this.pager.fetchNodes({});
+            }
+        }, 200);
```

Comment:
@bitwiseman, @ cliffmeyers this made significant improvement in our env.
consider even to increase from 200 to 500-1000